packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -15,40 +15,95 @@
  * limitations under the License.
  */
 
-import { DocumentData, Query, queryEqual } from './reference';
+import { CompositeFilterOpEnum } from '../protos/firestore_proto_api';
+import { invokeRunAggregationQueryRpc } from '../remote/datastore';
+import { hardAssert } from '../util/assert';
+import { cast } from '../util/input_validation';
+import { getDatastore } from './components';
+import { Firestore } from './database';
+import { Query, queryEqual } from './reference';
+import { LiteUserDataWriter } from './reference_impl';
 
-export class AggregateQuery<T = DocumentData> {
+/**
+ * A `AggregateQuery` computes some aggregation statistics from the result set of
+ * a base `Query`.
+ */
+export class AggregateQuery {
   readonly type = 'AggregateQuery';
-  readonly query: Query<T>;
+  /**
+   * The query on which you called `countQuery` in order to get this `AggregateQuery`.
+   * Query type is set to unknown to avoid error caused by query type converter.
+   * might change it back to T after testing if the error do exist or not
+   */
+  readonly query: Query<unknown>;
 
   /** @hideconstructor */
-  constructor(query: Query<T>) {
+  constructor(query: Query<unknown>) {
     this.query = query;
   }
 }
 
+/**
+ * A `AggregateQuerySnapshot` contains results of a `AggregateQuery`.
+ */
 export class AggregateQuerySnapshot {
   readonly type = 'AggregateQuerySnapshot';
   readonly query: AggregateQuery;
 
   /** @hideconstructor */
-  constructor(query: AggregateQuery, readonly _count: number) {
+  constructor(query: AggregateQuery, private readonly _count: number) {
     this.query = query;
   }
 
+  /**
+   * @return The result of a document count aggregation. Returns null if no count aggregation is
```

Comment:
Can you double-check that @return is a valid documentation string? Take a look at some of our other classes and see how they document the return values.

Reply:
in web SDK,some files used @return, some used @returns. 
based on JSDoc, @returns is the official tag, and @return is a valid synonym.

i will change this to @returns to follow the majority usage in web SDK